README.md

```diff
@@ -35,23 +35,24 @@ From Administrator's point of view, every Refund request results in creating two
     composer require sylius/refund-plugin
     ```
     
-    > Remember to allow not official recipes with `composer config extra.symfony.allow-contrib true` or during plugin installation process
+    > Remember to allow community recipes with `composer config extra.symfony.allow-contrib true` or during plugin installation process
 
-2. Copy migrations from `vendor/sylius/refund-plugin/migrations/` to your migrations directory (e.g. `src/Migrations`)
-and run `bin/console doctrine:migrations:migrate`
+2. Copy plugin migrations to your migrations directory (e.g. `src/Migrations`) and apply them to your database:
 
-3. Copy templates from `vendor/sylius/refund-plugin/src/Resources/views/SyliusAdminBundle/`
-to your templates directory (e.g `templates/bundles/SyliusAdminBundle/`)
+    ```bash
+    cp -R vendor/sylius/refund-plugin/migrations/* src/Migrations
+    bin/console doctrine:migrations:migrate
+    ```
 
-4. Clear cache:
+3. Copy Sylius templates overridden in plugin to your templates directory (e.g `templates/bundles/`):
 
     ```bash
-    bin/console cache:clear
+    cp -R vendor/sylius/refund-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
```

Comment:
Let's put `mdkir -p templates/bundles/SyliusAdminBundle` before `cp` to make sure that directory exists :)

Reply:
Done 🌮